pydantic/class_validators.py

```diff
@@ -42,6 +42,7 @@ def __init__(
 ROOT_KEY = '__root__'
 VALIDATOR_CONFIG_KEY = '__validator_config__'
 ROOT_VALIDATOR_CONFIG_KEY = '__root_validator_config__'
+SKIP_ON_FAILURE_KEY = '__validator_skip_on_failure__'
```

Comment:
better to move this into `Validator` rather than add another attribute, you'll need to include a comment saying that currently this only applies to root validators.

Reply:
See the change, I'm not sure I understood you correctly.

Reply:
The changed version looks good to me